source/sass/buyers-guide/views/product.scss

```diff
@@ -84,6 +84,14 @@
   h4 {
     font-family: "Nunito Sans";
   }
+
+  @import "../creep-face-emoji";
```

Comment:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/2896608/50182724-fa6ffe80-02c4-11e9-9ec3-a88b0033aabc.png">


We need to move this `@import` line to top of the page. We shouldn't include `@import` in a declaration block as what we wanna do here is to inherit and override the existing CSS rules from `creep-face-emoji.scss` and not declare the same rules for `.product-detail .creep-face-emoji` -- we should keep specificity low.

Reply:
As I read the Sass documentation, this is the way you should do an import if you need a specific class imported, so as far as I understood, this would have added the `.creep-face-emoji` class at this point in the product-detail stylesheet tree. From your screenshot, that is exactly what it does, so I'm not sure I understand: I _intended_ for this specificity, so that the base class definition, and its overrides, happen in exactly the same place.